integration_tests/src/main/python/orc_test.py

```diff
@@ -182,7 +182,7 @@ def test_orc_fallback(spark_tmp_path, read_func, disable_conf):
     with_cpu_session(
             lambda spark : gen_df(spark, gen).write.orc(data_path))
     assert_gpu_fallback_collect(
-            lambda spark : reader(spark).select(f.col('*'), f.col('_c2') + f.col('_c3')),
+            lambda spark : reader(spark).select(f.col('*'), f.col('_c2'), f.col('_c3')),
```

Comment:
"*" already cover column "_c2" and "_c3".  We can try to cast them to long and execute the "add" to avoid the overflow.